ui/dashboards/src/context/index.ts

```diff
@@ -14,4 +14,5 @@
 export * from './DashboardProvider';
 export * from './DatasourceStoreProvider';
 export * from './TemplateVariableProvider';
+export * from './SnackbarProvider';
```

Comment:
This probably makes sense to go into `@components` package rather than Dashboarding